cmsBuild

```diff
@@ -6,11 +6,13 @@ from os.path import abspath, join, exists, isdir, basename, dirname
 from os import popen, getenv, symlink, listdir, readlink, unlink, getpid
 from os import chdir, getcwd, environ, walk, sep, kill
 from getpass import getuser
+from sched import scheduler
```

Comment:
can you rename it so that one does not get it confused with our https://github.com/cms-sw/pkgtools/blob/V00-34-XX/scheduler.py